### DISCUSSION TITLE
關於功能的疑問

### DISCUSSION THREAD

**Discussion by NeKoOuO** [EhTagTranslation/EhSyringe#1239] [Q&A]
**從[Tachiyomi過來的](https://github.com/jobobby04/TachiyomiSY)而知道了這個項目，逛了一圈有兩個問題想解惑一下**
- 第一個是這個項目目前支持[**繁簡轉換**](https://github.com/EhTagTranslation/Database/issues/40#issue-759268569)嗎？或者繁轉簡搜索或者類似的功能，有支持到繁體的都算
- 第二是這個資料庫能夠支持其他漫畫軟體的導入嗎？例如這個老兄提到的[**Feature Request**](https://github.com/jobobby04/TachiyomiSY/issues/997#issue-2072657730)

**Comment by OpportunityLiu** [comment]
> * 第一個是這個項目目前支持[**繁簡轉換**](https://github.com/EhTagTranslation/Database/issues/40#issue-759268569)嗎？或者繁轉簡搜索或者類似的功能，有支持到繁體的都算

你可以试试opencc，但是效果并不怎么样



> * 第二是這個資料庫能夠支持其他漫畫軟體的導入嗎？例如這個老兄提到的[**Feature Request**](https://github.com/jobobby04/TachiyomiSY/issues/997#issue-2072657730)

这个跟本项目无关
